### PR TITLE
fix(GlobalHeaderItems): add new tab true to external links

### PR DESCRIPTION
**Pull request by saghdaey**
### Overview
We had missed adding "newTab: true" to a few of the app header items that should open in new tabs

all external links should open in a new tab 
ex. forums, chapters, blog, chat 

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

File: packages/gamut-labs/src/experimental/GlobalHeader/GlobalHeaderItems.tsx [hunk 1]
```diff
@@ -249,6 +252,7 @@ const profileCustomerSupport: AppHeaderLinkItem = {
 const profileReportBug: AppHeaderLinkItem = {
   id: 'report-bug',
   href: 'https://codecademy.atlassian.net/servicedesk/customer/portal/9',
+  newTab: true,
```
Comment (rogervera) [inline packages/gamut-labs/src/experimental/GlobalHeader/GlobalHeaderItems.tsx:255]: Sweet

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@10.0.1-alpha.229795.0<br/>@codecademy/styleguide@26.0.1-alpha.229795.0<br/></pre>

**Review by rogervera — APPROVED** [review]
Nit: Can we rename the PR to `fix(GlobalHeaderItems)`. 

https://github.com/Codecademy/client-modules#pr-title-format
Especially since the scope is just this.

LGTM!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@10.0.2-alpha.562d9d.0<br/>@codecademy/styleguide@26.0.2-alpha.562d9d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://602714941e69bd008e83a54c--gamut-preview.netlify.app>https://602714941e69bd008e83a54c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/602714941e69bd008e83a54c>Deploy Logs</a></p>

**Review by codecaaron — APPROVED** [review]

**Pull request merged**